lib/hubspot/meeting.rb

```diff
@@ -7,11 +7,42 @@ class Meeting
     #
     # {https://developers.hubspot.com/docs/api/crm/meetings}
     #
-    CREATE_MEETING_PATH    = '/crm/v3/objects/meetings'
+    MEETINGS_PATH          = '/crm/v3/objects/meetings'
     MEETING_PATH           = '/crm/v3/objects/meetings/:meeting_id'
+    MEETING_SEARCH_PATH    = '/crm/v3/objects/meetings/search'
     ASSOCIATE_MEETING_PATH = '/crm/v3/objects/meetings/:meeting_id/associations/Contact/:contact_id/meeting_event_to_contact'
 
+    BASE_PROPERTIES = %w[hubspot_owner_id hs_meeting_title hs_meeting_body hs_meeting_start_time hs_meeting_end_time].freeze
+
     class << self
+      def all(opts = {})
+        options = { properties: BASE_PROPERTIES.join(','), **opts.compact }
+        response = Hubspot::Connection.get_json(MEETINGS_PATH, options)
+        meetings = response['results'].map { |result| new(result) }
+
+        {
+          meetings: meetings,
+          after: response.dig('paging', 'next', 'after')
+        }
+      end
+
+      def find(id)
+        response = Hubspot::Connection.get_json(MEETING_PATH, { meeting_id: id, properties: BASE_PROPERTIES.join(',') })
+        new(response)
+      end
+
+      def find_by_contact(contact_id)
```

Comment:
Should we allow to add more filters as param ?

Reply:
YAGNI for now. Just like pagination. I didn't implement pagination for the `find_by_contact` function for now.